schemas/panels/time-series/time-series.cue

```diff
@@ -22,6 +22,7 @@ import (
 #legend: {
 	position: "Bottom" | "Right"
 	mode?:    "List" | "Table"
+	size?: "Small" | "Medium"
```

Comment:
FYI: some of the mocks include a "Large" size, but @foxbat07 suggested leaving it out based on some discussion of screen real estate, responsive behavior, etc.